doc/fileexplorer.rst

```diff
@@ -1,23 +1,84 @@
+If you right-click over it, you will see a context menu that allows you to access a number of functions, including running scripts; creating, renaming, moving, deleting files; and opening them in your computer's file manager.
+
+.. image:: images/files/files-context-menu.png
+   :alt: Spyder Files pane showing context menu
+
+You can copy and paste one or several files to and from the pane, and copy their absolute or relative paths to the clipboard as text.
+If copying the paths for multiple files, they will be automatically formatted so you can paste them directly into a Python list.
+
+.. image:: images/files/files-copy-path.gif
+   :alt: Spyder Files pane showing copying absolute path to Editor
+
+Additionally, the :guilabel:`Files` pane allows you to perform basic operations with ``git``, like commiting your changes and browsing the repository a given file or folder is part of.
+
+.. image:: images/files/files-commit.gif
+   :alt: Spyder Files pane showing commiting change of file
+
+
+
+============
+Options menu
+============
+
+The options menu in the top right of the :guilabel:`Find` pane offers several ways to customize how your files are displayed.
+
+By default, the option :guilabel:`Show all files` is activated, which displays the contents of your working directory without filtering. 
+However, if you deactivate this, the pane will only show files matching the patterns set under :guilabel:`Show filenames with these extensions...`.
+
+.. image:: images/files/files-filters.gif
+   :alt: Spyder Files pane showing filtering files
+
+You can also activate the :guilabel:`Show hidden files` option, which will display files that are invisible by default in your operating system.
+
+You can change which file attributes you want to see by hiding or displaying the :guilabel:`Type`, :guilabel:`Size` and :guilabel:`Date Modified` columns using the corresponding menu options.
+
+.. image:: images/files/files-columns-display.png
+   :alt: Spyder Files pane showing columns checked and shown
+
+The menu also gives you the option to open files and directories with a single- instead of a double-click, to suit your preferences.
+
+
+
+=================
+File associations
+=================
+
+You can associate different external applications with specific file extensions they can open.
+Under the :guilabel:`File associations` tab of the :guilabel:`Files` preferences pane, you can add file types and set the external program used to open each of them by default. 
+
+.. image:: images/files/files-associations.gif
+   :alt: Spyder Files pane showing files associations
+
+Once you've set this up, files will automatically launch in the associated application when opened from Spyder's :guilabel:`Files` pane.
+Additionally, when you right-click a file, you will find an :guilabel:`Open with...` option that will allow you to select from the applications associated with this extension.
+
+.. image:: images/files/files-associations-open.gif
```

Comment:
There appeared to be several seconds of black slug at the end of this GIF. Otherwise, it looked good.

Reply:
Agreed.

Reply:
Agreed

Reply:
Looks good now